scripts/zones/Giddeus/mobs/Hoo_Mjuu_the_Torrent.lua

```diff
@@ -3,11 +3,24 @@
 --  NM:  Hoo_Mjuu_the_Torrent
 -----------------------------------
 
+
+-----------------------------------
+-- onMobFight Action
+-----------------------------------
+
+function onMobFight( mob, target )
+    if (mob:getHPP() < 30 and mob:getLocalVar("benediction") == 0) then
+        mob:useMobAbility(689);
+        mob:setLocalVar("benediction", 1);
+    end
+end;
+
```

Comment:
IIRC doesn't retail trigger it at a random-ish hp% ? The following snippet is an example of how to do that: 
```lua
-----------------------------------
-- onMobSpawn Action
-----------------------------------

function onMobSpawn(mob)
    mob:setLocalVar("benedictionTrigger", math.random(10,60));
end;

-----------------------------------
-- onMobFight Action
-----------------------------------

function onMobFight( mob, target)
    local hpTrigger = mob:getLocalVar("benedictionTrigger");
    if (mob:getHPP() <= hpTrigger and mob:getLocalVar("benediction") == 0) then
        mob:useMobAbility(689);
        mob:setLocalVar("benediction", 1);
    end
end;
```

I'll prolly wrap the entire process of setting up and triggering 2hrs into a listener thing eventually.

Reply:
Most of my code is just pulled from other code that is already implemented:
https://github.com/DarkstarProject/darkstar/blob/7c03e7f96a9616fd2e0eac61bdc6e75c40931159/scripts/zones/Beadeaux/mobs/Ge_Dha_Evileye.lua#L19
I'm simply trying to follow the examples already given to fill in the blanks of what's still missing.

Reply:
In other words *"don't ask me to improve this, someone else can take it or leave it"* ?

Reply:
I'm sorry, that's now how I mean it, and I don't mean to take my bad day out on you. I would have PM you on that but I can't... sigh

Reply:
I'm just a leeetle dissapoint, but life goes on

Reply:
no harm done